roles/edpm_telemetry/tasks/deploy_ceilometer.yml

```diff
@@ -30,6 +30,18 @@
   tags:
     - edpm_telemetry
 
+- name: Configure ceilometer user and group on the host
+  ansible.builtin.import_role:
+    name: edpm_users
+  vars:
+    edpm_users_users:
+      # 42405 is matching with the uid and gid created by kolla in the ceilometer containers
+      - {"name": "ceilometer", "uid": "42405", "gid": "42405", "shell": "/sbin/nologin", "comment": "ceilometer user", "groups": "libvirt"}
+    edpm_users_extra_dirs: []
+  tags:
+    - edpm_users
+    - edpm_telemetry
```

Comment:
i dont see any molecule test for this
feel like adding a quick assert to ensure that this is created properly?